config/alexsmobs/komodo_dragon_spawns.json

```diff
@@ -1,16 +1,40 @@
 {
-  "biomes": [
-    [
-      {
-        "type": "BIOME_DICT",
-        "negate": false,
-        "value": "overworld"
-      },
-      {
-        "type": "BIOME_DICT",
-        "negate": false,
-        "value": "mesa"
-      }
+    "biomes": [
+        [
+            {
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "overworld"
+            },
+            {
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "mesa"
+            }
+        ],
+        [
+            {
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "overworld"
+            },
+            {
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "savanna"
+            }
+        ],
+        [
+            {
+                "type": "BIOME_DICT",
+                "negate": false,
+                "value": "overworld"
+            },
+            {
+                "type": "BIOME_CATEGORY",
+                "negate": false,
+                "value": "desert"
+            }
+        ]
```

Comment:
You can merge these into the same array and save a little size :)

Reply:
Ah cool. Wasn't too sure so I just copied from one of the others